src/vorta/borg/_compatibility.py

```diff
@@ -0,0 +1,24 @@
+from pkg_resources import parse_version
+
+MIN_BORG_FOR_FEATURE = {
+    'REPOKEY_BLAKE2': parse_version('1.1.4'),
```

Comment:
blake2 since 1.1.0.

Reply:
guess the key should be rather named 'BLAKE2' (it applies to repokey and also to keyfile modes).